app/views/govuk_publishing_components/components/_document_list.html.erb

```diff
@@ -5,13 +5,14 @@
   margin_top_class = " gem-c-document-list--top-margin" if local_assigns[:margin_top]
   margin_bottom_class = " gem-c-document-list--bottom-margin" if local_assigns[:margin_bottom]
   title_with_context_class = " gem-c-document-list__item-title--context"
+  underline_class = " gem-c-document-list--no-underline" if local_assigns[:remove_underline]
 
   brand ||= false
   brand_helper = GovukPublishingComponents::AppHelpers::BrandHelper.new(brand)
 %>
 <% if items.any? %>
   <% unless within_multitype_list %>
-    <ol class="gem-c-document-list<%= margin_bottom_class %><%= margin_top_class %>">
+    <ol class="gem-c-document-list<%= margin_bottom_class %><%= margin_top_class %><%= underline_class %>">
```

Comment:
Just a readability suggestion - do you think it's worth pushing these to a single variable to then be used in the class of the `ol` element? 

For example:
```ruby
list_class = ['gem-c-document-list', underline_class, margin_top_class, margin_bottom_class]
```

then

```ruby
<ol class="<% list_class %>">
```

Reply:
I did, but I was hoping to get away with just one more... okay, I'll change it 😁

Reply:
Changed.